extra/signed-artifact-client-testing/docker-compose.signed-client.yml

```diff
@@ -5,7 +5,7 @@ services:
     # mender-client
     #
     mender-client:
-        image: mendersoftware/mender-client-qemu:master
+        image: mendersoftware/mender-client-qemu:pr
```

Comment:
This is added on top of the other docker compose files, right? In that case `image` should not be specified at all, you should just rely on the existing one.